bin/common.ml

```diff
@@ -851,6 +857,9 @@ module Builder = struct
             [ "display-separate-messages" ]
             ~doc:"Separate error messages with a blank line.")
     in
+    if Option.is_none stats_trace_file && stats_trace_extended then
+      User_error.raise
+        [ Pp.text "Thou shalt not pass --trace-extended without --trace" ];
```

Comment:
I would suggest saying this in a less commanding manner :)

Something like:
```
--trace-extended must be passed together with --trace
```
for consistency with our other messages.

Reply:
@rgrinberg?

Reply:
I'm not a huge fan of adding religious references either, so I second @Alizter's suggestion.

Reply:
It's not the reference that bothers me but rather keeping messages understandable to users, especially for those who don't have English as a first language. I'm happy to improve this message later in another PR if you prefer however.